packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1255,7 +1256,7 @@ Link: <https://${yourOktaDomain}/api/v1/users?after=00ud4tVDDXYVKPXKVLCO&limit=2
 #### Find Users
 
 
-Finds users who match the specified query
+Finds users who match the specified query.  To ensure optimal performance, we recommend using a [search parameter](#list-users-with-search) instead.
```

Comment:
Do we have a standard of one space or two spaces following a period? Applies here and in several other places. 

Beyond that, is it typical to use "we" in the documentation or should we be rephrasing this more like "...The [search parameter](#list-users-with-search) is preferred whenever possible

Reply:
"We" shows up often enough in the doc.  And I've done two spaces forever, and I'm pretty sure most modern rendering engines automatically shrink the 'space between sentences' to what they think anyway.

Reply:
From a docs perspective, we try not to use first-person (we) in our docs, although I'm sure there are lots of examples. If we do have a recommendation, we tend to use "Okta recommends" rather than "We".

In terms of periods, we use single space after a period, although Vuepress might render it as one-space no matter what.